lib/src/util/safe_render_manager/safe_render_manager_helper.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:over_react/over_react.dart';
+
+// ignore: uri_has_not_been_generated
+part 'safe_render_manager_helper.over_react.g.dart';
+
+/// A component that allows for safe unmounting of its single child by waiting for state changes
+/// sometimes queued by ReactJS to be applied.
+@Factory()
+UiFactory<SafeRenderManagerHelperProps> SafeRenderManagerHelper =
+// ignore: undefined_identifier
+_$SafeRenderManagerHelper;
+
+@Props()
+class _$SafeRenderManagerHelperProps extends UiProps {
+  @requiredProp
+  ReactElement Function() getInitialContent;
```

Comment:
Currently the master branch has to remain Dart 1 and Dart 2 friendly.  IIRC, inline function typedef syntax like this causes problems in Dart 1